src/bin/pg_autoctl/pgsetup.c

```diff
@@ -160,11 +156,10 @@ pg_setup_init(PostgresSetup *pgSetup,
 	}
 	else
 	{
-		char *pguser = getenv("PGUSER");
-
-		if (pguser)
+		if (!get_env_copy_with_fallback("PGUSER", pgSetup->username, NAMEDATALEN, ""))
```

Comment:
I think we need a comment to explain why it's okay to use "" as the fallback here, rather than NULL and have an error.